src/authentication/messages.json

```
@@ -0,0 +1,24 @@
+{
+  "authentication": {
+    "title": "Authentication",
+    "flows": "Flows",
+    "flowName": "Flow name",
+    "usedBy": "Used by",
+    "buildIn": "Build in",
```

```suggestion
    "buildIn": "Built-in",
```

Maybe should change the key as well.  The English expression is "Built-in"